### PR TITLE
Add Welcome Content in View

### PR DESCRIPTION
**Pull request by hyunsik-yoon**
This adds welcome content in our view, which asks end-user to do
some initial settings.

parent: #330 
ref: https://code.visualstudio.com/api/references/contribution-points#contributes.viewsWelcome

ONE-vscode-DCO-1.0-Signed-off-by: Hyun Sik Yoon <hyunsik.yoon.1024@gmail.com>

### CODE REVIEW

**Comment by hyunsik-yoon** [comment]
After this PR, ONE-vscode shows the following when user clicks `ONE` icon in sidebar.

image

File: package.json [hunk 1]
```diff
@@ -39,6 +43,13 @@
         }
       ]
     },
+    "viewsWelcome": [
+      {
+        "view": "FileTreeView",
+        "contents": "Open folder where model files are stored. Models files are one of the following format:\n- `*.tflite` or saved model for Tensorflow​\n- `*.onnx` or `*.pth` for PyTorch\n[Open Folder](Command:onevscode.fileTreeView-open)\nNext, register already-installed toolchain or install a new toolchain. You can do this later.\n[Register Installed Toolchain](Command:onevscode.register-installed-toolchain)\nor\n[Install New Toolchain](Command:onevscode.install-new-toolchain)\n",
+        "when": "config.one-vscode.projectRootDirectory == ''"
```
Comment (hyunsik-yoon) [inline package.json:50]: For the reason why `config.` is attached to `one-vscode.projectRootDirectory` setting name,
please refer to the following:
- https://code.visualstudio.com/api/references/when-clause-contexts#check-a-setting-in-a-when-clause

**Comment by hyunsik-yoon** [comment]
CI passed! 📯

File: package.json [hunk 2]
```diff
@@ -39,6 +43,13 @@
         }
       ]
     },
+    "viewsWelcome": [
+      {
+        "view": "FileTreeView",
+        "contents": "Open folder where model files are stored. Models files are one of the following format:\n- *.tflite or saved model for Tensorflow​\n- *.onnx or *.pth for PyTorch\n[Open Folder](Command:onevscode.fileTreeView-open)\nNext, register already-installed toolchain or install a new toolchain. You can do this later.\n[Register Installed Toolchain](Command:onevscode.register-installed-toolchain)\nor\n[Install New Toolchain](Command:onevscode.install-new-toolchain)\n",
```
Comment (llFreetimell) [inline package.json:49]: `.pb` is omitted intentionally?

Comment (hyunsik-yoon) [reply package.json:49]: I will add. Thanks. 
@llFreetimell and @mhs4670go, please let me know if ONE supports more extensions.

Comment (llFreetimell) [reply package.json]: @hyunsik-yoon Sorry for bothering you :( `keras_model` is also supported...

File: package.json [hunk 3]
```diff
@@ -11,12 +11,16 @@
...
   ],
   "activationEvents": [
+    "onView:FileTreeView",
     "onCommand:onevscode.build",
     "onCommand:onevscode.import",
     "onCommand:onevscode.json-tracer",
     "onCommand:onevscode.configuration-settings",
     "onCommand:onevscode.toggle-codelens",
     "onCommand:onevscode.circle-tracer",
+    "onCommand:onevscode.fileTreeView-open",
```
Comment (jyoungyun) [inline package.json:21]: How about adding this changes when you implement a real operation?

File: package.json (hunk 2 above)
Comment (hyunsik-yoon) [reply package.json]: Thanks. I added Keras model into the description string.

File: package.json (hunk 3 above)
Comment (hyunsik-yoon) [reply package.json:21]: Removing the line seems not good since we have a button that triggers the event.
Instead, I added empty event handlers. PTAL.

**Comment by hyunsik-yoon** [comment]
License verification showed different result. (one with no error, but another with 6 warnings)
/cc @llFreetimell

**Comment by hyunsik-yoon** [comment]
All of WTFPL, BSD-2-Clause, MIT, Apache 2 have no problem. Therefore I will ignore these warnings.

**Comment by llFreetimell** [comment]
@hyunsik-yoon 

I cross-checked above packages and there is no problem :)
Sorry for checker bug... you can ignore them!

**Review by llFreetimell — APPROVED** [review]
LGTM

**Comment by hyunsik-yoon** [comment]
@mhs4670go, @jyoungyun PTAL

**Review by llFreetimell — APPROVED** [review]
LGTM

**Comment by hyunsik-yoon** [comment]
@mhs4670go, @jyoungyun PTAL

File: src/extension.ts [hunk 4]
```diff
@@ -29,6 +29,15 @@ import {Utils} from './Utils';
 export function activate(context: vscode.ExtensionContext) {
   console.log('one-vscode activate OK');
 
+  let fileTreeViewOpenCmd =
```
Comment (jyoungyun) [inline src/extension.ts:32]: 👍

**Review by jyoungyun — APPROVED** [review]
LGTM

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**